src/api/Corona.js

```diff
@@ -0,0 +1,24 @@
+const cheerio = require('cheerio');
+const api = require('./api');
+
+const dgsReports = 'https://covid19.min-saude.pt/relatorio-de-situacao/';
+
+const getReports = async () => {
+  const data = [];
+  const pageHtml = await api.getHtml(dgsReports);
+
+  const $ = cheerio.load(pageHtml);
+
+  $('.single_content ul li').each((i, elem) => {
+    data.push({
+      link: $(elem).find('a').prop('href'),
+      title: $(elem).text(),
+    });
+  });
+
+  return data;
```

Comment:
No need for intermediate variable `data` and pushing into it. `.map()` was built to deal with these cases.

```js
  return $('.single_content ul li').map((i, elem) => ({
    link: $(elem).find('a').prop('href'),
    title: $(elem).text(),
  });
```

Reply:
Fixed in https://github.com/vostpt/bot/pull/154